src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -210,16 +210,18 @@ const FailedTestsTable = () => {
       // eslint-disable-next-line camelcase
       test_suites: testSuites as string[],
       parameter: queryParams?.parameter as TestResultsFilterParameterType,
-      history: queryParams?.historicalTrend as MEASUREMENT_INTERVAL_TYPE,
+      interval: queryParams?.historicalTrend as MEASUREMENT_INTERVAL_TYPE,
     },
     opts: {
       suspense: false,
     },
   })
 
+  // Only show flake rate column when on default branch for pro / enterprise plans or public repos
   const hideFlakeRate =
-    (isTeamPlan(testData?.plan) || isFreePlan(testData?.plan)) &&
-    testData?.private
+    ((isTeamPlan(testData?.plan) || isFreePlan(testData?.plan)) &&
```

Comment:
had some trouble parsing out this conditional but seems to align with what the comment says anyway so prob fine

Reply:
I see what you mean; probably worth breaking it out into two separate variables and then having hideFlakeRate be an || of those variables 🤔 

Ill make a note of that next time for next time I'm in this file